### ISSUE TITLE
Model re-download after we changed the default to our own

### ISSUE THREAD

**Issue by TomasTomecek** [fedora-copr/logdetective#60]
bd5c0cddc2b82f5b0c841d687bc6ea7765c8d2c6

```
$ logdetective -v some.log
ggml-model-Q4_K_S.gguf: 100%|█████████████████████████████| 4.14G/4.14G [05:42<00:00, 10.7MB/s]
```

I was frustrated I had to download the model again.

Now that I look how HF stores models locally, it seems it has to be downloaded anyway:
```
$ ll ~/.cache/huggingface/hub/models--fedora-copr--Mistral-7B-Instruct-v0.2-GGUF/snapshots/f8d679d67ad3e551f131ceff60a673d3a6ff4286/ggml-model-Q4_K_S.gguf
lrwxrwxrwx. 1 tt tt 76 sep  3 09:58 /home/tt/.cache/huggingface/hub/models--fedora-copr--Mistral-7B-Instruct-v0.2-GGUF/snapshots/f8d679d67ad3e551f131ceff60a673d3a6ff4286/ggml-model-Q4_K_S.gguf -> ../../blobs/be98b5d472de622d4d413550e00d04b06c35e7bec087ddb1f10837687c735606

$ ll ~/.cache/huggingface/hub/models--TheBloke--Mistral-7B-Instruct-v0.2-GGUF/snapshots/3a6fbf4a41a1d52e415a4958cde6856d34b2db93/mistral-7b-instruct-v0.2.Q4_K_S.gguf
lrwxrwxrwx. 1 tt tt 76 aug  8 10:53 /home/tt/.cache/huggingface/hub/models--TheBloke--Mistral-7B-Instruct-v0.2-GGUF/snapshots/3a6fbf4a41a1d52e415a4958cde6856d34b2db93/mistral-7b-instruct-v0.2.Q4_K_S.gguf -> ../../blobs/1213e19b3e103932fdfdc82e3b6dee765f57ad5756e0f673e7d36514a5b60d0a
```

It would be amazing if the filename at least said it's mistral.

**Comment by jpodivin** [comment]
I agree that it's unfortunate the model is duplicated this way. But it's actually necessary. We can compare models layer by layer, to ascertain they are same or not, but that take time, hash is fast, but it will be affected by even minute changes in metadata, which can happen between llama.cpp releases. 

As far as HF is concerned, the checking isn't worth the time. User requested this file, from this repo and that's what they get. The name of the model is still in the directory on level higher. I thought it's enough, but if you want we can rename it.

**Comment by TomasTomecek** [comment]
That's true, the model name is in the path, though it's not immediately obvious.

The reason I got confused was when ran the `logdetective` command, the first line I saw was about downloading `ggml-...`. At first I thought we've completely changed to a different model. It wasn't obvious to me what this "ggml" is.

It may be just my own problem because I can see under the hood.

Something like this would solve my problem:
```
$ logdetective -v some.log
INFO: Starting download of the default model 'mistral-7b-Q4_K_S'
ggml-model-Q4_K_S.gguf: 100%|█████████████████████████████| 4.14G/4.14G [05:42<00:00, 10.7MB/s]
```

But again, I'm not sure if our users actually care about this.
